azure-pipelines.yml

```diff
@@ -173,6 +173,11 @@ stages:
             displayName: Publish the tool artifact
             condition: and(succeeded(), eq(variables['_BuildConfig'], 'Debug'))
 
+          - publish: $(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)/Shipping/Microsoft.DotNet.XHarness.CLI.1.0.0-ci.nupkg
```

Comment:
I test with one flavor of the mac one because the package made should be the same and this runs MUCH faster than the buildpool.* machines